### DISCUSSION TITLE
Entities have suddenly gone missing

### DISCUSSION THREAD

**Discussion by LordLinhey** [tsightler/ring-mqtt#761] [Support Q&A]
Suddenly the entities have gone missing from the 2 services provided by MQTT and I really can't figure this out, I have tried restaring each of the integrations related to Ring  (MQTT, Ring-MQTT and Ring).  To illustrate my issue I have included some screenshots.
I hope someone can help me with this. In particular I use the alarm mode in my automations, which now of course don't work.
![device info](https://github.com/tsightler/ring-mqtt/assets/77053272/d22d8620-fccb-4b66-ab84-8330c4b4ef11)
![mqtt devices](https://github.com/tsightler/ring-mqtt/assets/77053272/9560f193-861e-4f93-93dc-fb7c8b38bbb2)
![Ring mqtt Entities](https://github.com/tsightler/ring-mqtt/assets/77053272/aa9ece05-a513-476c-aee3-4b284a94f16f)
![Ring-MQTT log](https://github.com/tsightler/ring-mqtt/assets/77053272/d2031ed9-0a92-4c23-93eb-51aaa1b2b9b8)
![unavailable](https://github.com/tsightler/ring-mqtt/assets/77053272/332eb35b-e68f-4e93-b187-8b4a8f86f1ae)

**Comment by GHaigh** [comment]
I am having exactly the same issue, I have tried forcing re-authentication a couple of times but that hasn't helped.  Anyone any ideas?

Greg

**Comment by LordLinhey** [comment]
Hi Greg,

I gave up trying to fix this the hard way and decided if it doesn't work
uninstall it

DO THIS

1 Uninstall the Mosquitto Broker and the Ring-MQTT integration
2 Restart Home Assistant  (IMPORTANT)

3 Install and setup Mosquitto Broker
4 Install and setup Ring-MQTT

5 Restart Home Assistant and create a full backup in  case this happens
again
6 Setup the integrations, (completely automatic)

It took under 10 minutes to complete, instead of faffing about with the
original installation

Hope it works for you
LL
